integration_testing/features/app-context/app-context-manage-sign-in-options.feature

```diff
@@ -50,11 +56,96 @@ Feature: Manage sign in options within app context
 			And I tap *Sign in*
 		Then I see learner <learner> *Learn* page
 
+	Scenario: Learner signs in directly by taping their username (multiple facilities)
+		Given there is more than one facility on the device
+			And that signing in without password is enabled in the *Facility Settings*
+			And <facility> has less then 16 users
+			And I see *Sign in if you have an existing account*
+		When I tap the *Sign in* button
+		Then I see the list of facilities
+		When I tap the <facility> button
+		Then I see a list of all the usernames
+		When I tap learner <learner> username
+		Then I see learner <learner> *Learn* page
+
+	Scenario: Sign in when password is required and more than 16 users in the facility (multiple facilities)
+		Given there is more than one facility on the device
+			And that signing in without password is disabled in the *Facility Settings*
+			And <facility> has more then 16 users
+			And I see *Sign in if you have an existing account*
+		When I tap the *Sign in* button
+		Then I see the list of facilities
+		When I tap the <facility> button
+		Then I see the the username input field
+		When I type learner <learner> username <username>
+			And I tap *Next*
+		Then I see the password input field
+		When I type learner <learner> password <password>
+			And I tap *Sign in*
+		Then I see learner <learner> *Learn* page
+
+	Scenario: Any subsequent session on Kolibri after first sign in (single facility; with password; less then 16 users)
+		Given I the device has only one facility
+			And I have used this device previously to sign in or create a new learner account
+			And that signing in without password is disabled in the *Facility Settings*			
+		When I tap the app icon to open Kolibri
+		Then I see *Sign into '<facility>'*
+			And I see a list of usernames
+		When I tap my <username>
+		Then I see *Sign into '<facility>' as '<username>'*
+			And I see the input field for entering my password
+			And I type my password
+			And I click on *Sign in*
+		Then I see my *Learn* page
+
+	Scenario: Any subsequent session on Kolibri after first sign in (multiple facilities; with password; less then 16 users)
+		Given I the device has with multiple facilities
+			And I have used this device previously to sign in or create a new learner account
+			And that signing in without password is disabled in the *Facility Settings*			
+		When I tap the app icon to open Kolibri
+		Then I see the *Select facility...*
+			And I see a list of facilities
+		When I tap to select <facility>
+		Then I see *Sign into '<facility>'*
+			And I see a list of usernames
+		When I tap my <username>
+		Then I see *Sign into '<facility>' as '<username>'*
+			And I see the input field for entering my password
+			And I type my password
+			And I click on *Sign in*
+		Then I see my *Learn* page
+
+	Scenario: Facility name will not be displayed on the sign in page if there is only one facility on the device and the use context is personal
```

Comment:
@nucleogenesis This scenario is verbatim from Riddhi's gherkins on Notion, but I still see the facility name when I install on Android device with **Quick setup** (which should be how the use is recognized as _personal_, correct?)

![DevTools - localhost:8080-en-user-_882](https://user-images.githubusercontent.com/1457929/88188920-bd09ea80-cc38-11ea-9f63-55f06996f6cb.png)

Reply:
Ok, issue already reported here: https://github.com/learningequality/kolibri/issues/7303#issuecomment-661272301